### DISCUSSION TITLE
HORDE -- undead legionnaire

### DISCUSSION THREAD

**Discussion by TomekPL-1** [secretcowlevel/DoomtrooperPublic#17] [Card Balance]
HORDE - Adding same warriors it to SQUAD merge them in to one warrior adding their F, S , A &V. When HORDE is wounded opponent earn VP only for one HORDE member and lower stats of the HORDE, SLAYER = 2 wounds and kills 2 members of the HORDE. (cards affecting warriors like REASSIGMENT or BANISHMENT affect only one member of the HORDE)

Undead Legionnaire
NOEQUIP, UNLIMITED, HORDE.

Nepharite Warlord
NEPHARITE At the end of your turn add one UNDEAD LEGIONNAIRE from your discard pile in to the SQUAD

Necromutant
If Nepharite Warlord is in your SQUAD you gain 1 additional STANDARD ACTION.

Necromutant Frenzy
INFLUENCE. For every Necromutant in your SQUAD your UNDEAD LEGIONNAIRE can use one STANDARD ACTION as TACTICAL ACTION

Centurion
If you have 3 or more UNDEAD LEGIONNAIRE in your discard pile every time you add UNDEAD LEGIONNAIRE to your HORDE draw 1 card.

**Comment by TomekPL-1** [comment]
Justin have good idea to remove UNLIMITED from Undead Legionnaire and make 

Nepharite Warlord (Dark Legion) [R]
NEPHARITE. At the end of your turn create one UNDEAD LEGIONNAIRE TOKEN an add it to the SQUAD
F: 7 S: 4 A: 5 V: 7 (proposed stats)

**Reply by 21-ric** [reply]
We also have "portal of undeath" for bringing "undead legionnaire".

Portal Of Undeath (ALGEROTH) [R]
FACTION, DARKSYMMETRY
Bestow only on a NEPHARITE. After this Warrior kills an enemy, a random ALGEROTH Warrior with [V] 5 joins your side.

The original card :
Dark Symmetry • Dark Legion
Gift of Algeroth Nepharite only - Doomtrooper killed by the Nepharite are resurrected under your control. Place an Undead Legionnaires card from your collection into your Kohort. If you can't still discard the warrior.

**Reply by Kite80** [reply]
I agree with the original card text, it is called "portal of undead" for a reason and the cance to get any Algeroth warrior has changed it too much
